impeller/renderer/backend/metal/allocator_mtl.mm

```diff
@@ -212,11 +226,23 @@ static MTLStorageMode ToMTLStorageMode(StorageMode mode,
     }
   }
 
+#ifdef IMPELLER_DEBUG
+  if (desc.storage_mode != StorageMode::kDeviceTransient) {
+    debug_allocater_->Increment(desc.GetByteSizeOfBaseMipLevel());
```

Comment:
Does this mean we aren't tracking the size of the other mip levels?  That will give us quite a different result?

Reply:
Done